cmd/cephcsi.go

```diff
@@ -81,6 +81,13 @@ func init() {
 		"",
 		"list of kubernetes node labels, that determines the topology"+
 			" domain the node belongs to, separated by ','")
+	flag.BoolVar(&conf.EnableReadAffinity, "enable-read-affinity", false, "enable read affinity")
+	flag.StringVar(
+		&conf.CrushLocationLabels,
+		"crush-location-labels",
+		"",
+		"list of kubernetes node labels, that determines the"+
+			" crush location the node belongs to, separated by ','")
```

Comment:
CRUSH is an abbreviation so it should be capitalized in the option descriptions and documentation:
```suggestion
			" CRUSH location the node belongs to, separated by ','")
```